tests/legacy/ApiTest.php

```diff
@@ -583,26 +587,7 @@ public function testApiRssExtraWithoutUserInfo()
 		self::assertArrayHasKey('atom_updated', $result['$rss']);
 		self::assertArrayHasKey('language', $result['$rss']);
 		self::assertArrayHasKey('logo', $result['$rss']);
-	}
-
-	/**
-	 * Test the api_unique_id_to_nurl() function.
-	 *
-	 * @return void
-	 */
-	public function testApiUniqueIdToNurl()
```

Comment:
concerning all deleted tests:

Are they deprecated, because there is no similar functionality anymore, or are there no possible tests so far? (because lack of datasets e.g.)

Reply:
The deleted ones are for functions that don't exist anymore. When I had moved the functions then I just had commented out the tests.

Reply:
I see thx